fares-data-build-tool/create-data
# feat: CFD-306 Netex updates for prices per SOP

# Description

-   Netex updates for prices per SOP

# Testing instructions

-   Complete any Period or Flat Fare journey
-   When you get to the Sales Offer Package page tick multiple options
-   Change the prices for some of the options and remember the values
-   Generate the Netex and check that the price per SOP is included as the value 

# Type of change
